specification/netapp/resource-manager/Microsoft.NetApp/stable/2023-07-01/examples/CheckNameAvailability.json

```
@@ -0,0 +1,19 @@
+{
+  "parameters": {
+    "subscriptionId": "D633CC2E-722B-4AE1-B636-BBD9E4C60ED9",
+    "location": "eastus",
+    "api-version": "2023-07-01",
+    "body": {
+      "name": "accName",
+      "type": "netAppAccount",
```

```suggestion
      "type": "Microsoft.NetApp/netAppAccounts",
```
please update to this